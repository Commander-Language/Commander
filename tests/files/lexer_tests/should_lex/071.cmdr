// Test that print statements lex properly
print "Hello";
