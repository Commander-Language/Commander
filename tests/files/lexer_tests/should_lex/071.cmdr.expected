PRINT 'print' 2:1
STRINGVAL 2:7
[
STRINGLITERAL 'Hello' 2:8
]
SEMICOLON ';' 2:14
END_OF_FILE 3:1
